packages/client/src/app/pages/home/settings/settings.component.ts

```diff
@@ -194,4 +237,8 @@ export class SettingsComponent implements OnInit {
       location.reload();
     });
   }
+
+  submitContentFilter() {
+      this.authService.setContentFilter(this.setFilterFields.enableMature.value, this.setFilterFields.enableExplicit.value);
```

Comment:
Am I correct in assuming content filtering is set _both_ locally in a `contentFilter` cookie, _and_ server-side? (and when a user logs in, their `contentFilter` cookie gets populated?)

Reply:
ah, no, it's _only_ local. Okay. May want to store serverside in a later update

Reply:
something to consider, yeah.

the idea behind it only being stored locally was so users can have different filter settings depending on what browser/computer logged into. like, say, a user might want to have mature stories turned on at home but when they're just browsing at work they want that turned off.